README.md

```diff
@@ -57,10 +57,10 @@ Images
 
     ```sh
```

Comment:
isnt this obsolete when using the yaml file for the environment? shoulr just state that the environment should be setup with the yaml file instead